api/openapi.yaml

```diff
@@ -690,6 +690,8 @@ components:
           $ref: "#/components/schemas/ApiItem_Service"
         version:
           type: string
+        crunch42URL:
```

Comment:
In another file, you've got it spelt as `crunc42url`. I'd use `42crunch_URL` or `42crunch_url`. Whatever you prefer.

Reply:
I fixed the typo. I'm not a fan of using underscores for variable namees.

Reply:
You have

```go
Crunch42URL string `json:"crunch42url"`
```

in `server/go/model_api_item.go`.

and in `api/openapi.yaml` you have `crunch42URL`.

Does the casing make a difference?

Reply:
oh :D